docs/user/logs.md

```diff
@@ -0,0 +1,28 @@
+# Logs
+
+FlowForge presents log information in several different places depending on what you are interested in.
+
+## Project Logs
+
+Within the project view the logs tab displays the Node-RED log, this will contain information such as nodes being added, and errors relating to your flows.
```

Comment:
```suggestion
Within the project view the logs tab displays the Node-RED log. This will contain information such as nodes being added and errors relating to your flows.
```